apps/studio/schemas/organizer.ts

```diff
@@ -0,0 +1,34 @@
+import { HeartIcon } from "@sanity/icons";
+import { defineField, defineType } from "sanity";
+
+export default defineType({
+  name: "organizer",
```

Comment:
I think I'm still a bit confused about the separate organizer schema. I don't quite understand the advantages of this over consolidating it into a person schema.
I do vaguely recall you mentioning something about separating organizers from other people. We'd be able to do this **without** a separate schema. Sanity Studio is pretty customizable, so we could, for example, have a tab that's filtered to all organizers, or this years organizers.

Reply:
After discussion over Slack, we decided to rename the `organizer` schema to `person`. This was accomplished in [`babb28d`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/babb28de063249dfcad0d491a4467125cb60e11d),